### PR TITLE
Added AND_OR_XOR_Converter

### PR DESCRIPTION
**Pull request by Avishake007**
## Related Issue
Added AND, OR and XOR Converter

Closes: #312 

### Describe the changes you've made
Here I have added the html, css and js files of  AND,OR and XOR Converter

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
andorxor(1)

### CODE REVIEW

**Comment by Avishake007** [comment]
@urvashi-code1255  please check

**Comment by 991rajat** [comment]
@Avishake007 Good Work
- Add the project to script.js
- Squash the commits
> Let me know if you need any help

**Comment by Avishake007** [comment]
@991rajat  By adding  the project to script.js do you mean to renaming the file to script.js

**Comment by 991rajat** [comment]
Also squash the commits into one 
What squashing does is it combines multiple commits into single commit removing unnecessary commits and add descriptive message. what you need to do is to squash them into one
squash only your 6 commits.
https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc
Let me know if need any help

**Comment by Avishake007** [comment]
@991rajat I am facing problem in squashing the commits. Could you please tell the admin to squash and merge. Then I think it will be alright.

**Comment by 991rajat** [comment]
@Avishake007  DM me on discord.

File: script.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
 let projectData = [
+ {
+  projectname: 'And Or Xor Converter',
+  projectUrl: 'public/AND,OR,XOR Converter.html',
```
Comment (991rajat) [inline script.js:4]: - Don't insert comma and space in filename.
- Add gif to the project.

Fix it please

**Comment by 991rajat** [comment]
@Avishake007 
- Css is not linked properly.
- Project name is undefined.

Please fix it.

**Comment by Avishake007** [comment]
@991rajat  Kindly check

File: script.js [hunk 2]
```diff
@@ -4,6 +4,11 @@ let projectData = [
     projectImage: 'assets/GIFs/Tubelight-text.gif',
     projectUrl: 'public/Tubelight-text.html',
   },
+  {
+    projectname: 'And Or Xor Converter',
```
Comment (991rajat) [inline script.js:8]: Name is Undefined
Fix projectName

**Review by 991rajat — APPROVED** [review]
Good Work.

**Pull request merged**